CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 <!--## Unreleased-->
+- `build`: Update automatic `webcomponentsjs` polyfilling to move it and all affected elements following it into the body so that the `custom-elements-es5-adapter.js` can work properly in IE11. (See [#627](https://github.com/Polymer/polymer-cli/issues/627))
```

Comment:
The README is verrrrry out of date, will add for now and then start a new issue to update it